tests/meta/input.ml

```diff
@@ -1,9 +1,10 @@
 open Test_lib
 open Report
 
+let%meta identifier = "meta"
 let%meta stars = 3
 let%meta title = "Blimey! this is a really trivial exercise"
 let%meta short_description = "Identity function"
-let%meta authors = ["Me", "me@myself.fr"]
+let%meta authors = [("Me", "me@myself.fr")]
```

Comment:
This is not very elegant: why is it needed to wrap the pair with a list constructor?

Reply:
Oh I see: this is for author*s*. Can you define "let%meta author" too to avoid this disgusting type-related glinch?